app/experimenter/experiments/models.py

```diff
@@ -44,6 +44,9 @@ class Experiment(ExperimentConstants, models.Model):
     owner = models.ForeignKey(
         get_user_model(), blank=True, null=True, on_delete=models.CASCADE
     )
+    subscribers = models.ManyToManyField(
+        get_user_model(), blank=True, related_name="experiments"
```

Comment:
Change that to `related_name=subscribed_experiments` and update references in the templates